packages/create-block/lib/index.js

```diff
@@ -107,14 +113,30 @@ program
 
 					const filterOptionsProvided = ( { name } ) =>
 						! Object.keys( optionsValues ).includes( name );
-					const blockPrompts = getPrompts( pluginTemplate, [
-						'slug',
-						'namespace',
-						'title',
-						'description',
-						'dashicon',
-						'category',
-					] ).filter( filterOptionsProvided );
+
+					// Get the variant prompt first. This will help get the default values
+					const variantPrompt =
```

Comment:
When I provide the variant from the CLI, it asks again for the variant. We can address it seperately.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/43481.